src/routes/PrimaryLayout/PrimaryLayout.js

```diff
@@ -161,11 +161,11 @@ export default class PrimaryLayout extends Component {
         <div styleName={cx('sidebar', { hidden: (hasDetail || isMapViewPath(location.pathname)) })}>
           <Switch>
             <Route path={`/:context(c)/:slug${OPTIONAL_NEW_POST_MATCH}`} exact component={CommunitySidebar} />
-            <Route path={`/:context(c)/:slug/m/:personId/${OPTIONAL_NEW_POST_MATCH}`} component={MemberSidebar} />
+            {/* <Route path={`/:context(c)/:slug/m/:personId/${OPTIONAL_NEW_POST_MATCH}`} component={MemberSidebar} /> */}
```

Comment:
we are just getting rid of this no?

Reply:
I left this in in case you guys didn't like not having a sidebar. The layout seems a little strange in the "All Communities" context. Take a look, but either way though I'll remove it.

Reply:
yeah i agree it looks a little weird. hmm, not sure what it should be